.chalice/config.json

```diff
@@ -35,7 +35,7 @@
   "app_name": "dos-azul-lambda",
   "environment_variables": {
     "DATA_OBJ_INDEX": "fb_index",
-    "DEBUG": "False"
+    "DEBUG": "True"
```

Comment:
The original ticket specifies that this should be changed, and I think I told you that in person as well. I was wrong on both counts - `.chalice/config.json` does not need to be changed, since debug should always be disabled in production. Not sure why I thought it should be changed, sorry for the runaround...